singlestore/README.md

```diff
@@ -0,0 +1,85 @@
+# Agent Check: SingleStore
+
+## Overview
+
+This check monitors [SingleStore][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The SingleStore check is included in the [Datadog Agent][3] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Host" xxx -->
+
+#### Host
+
+##### Metric collection
+1. Edit the `singlestore.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your SingleStore performance data. See the [sample singlestore.d/conf.yaml][4] for all available configuration options.
+
+2. [Restart the Agent][5].
+
+<!-- xxz tab xxx -->
+<!-- xxx tab "Containerized" xxx -->
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying the parameters below.
+
+#### Metric collection
+
+| Parameter            | Value                                                      |
+|----------------------|------------------------------------------------------------|
+| `<INTEGRATION_NAME>` | `singlestore`                                                   |
+| `<INIT_CONFIG>`      | blank or `{}`                                              |
+| `<INSTANCE_CONFIG>`  | `{"host": "%%host%%", "port": "%%port%%", "username": "<USER>", "password": "<PASSWORD>"}`       |
+
+
+<!-- xxz tab xxx -->
+<!-- xxz tabs xxx -->
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `singlestore` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][7] for a list of metrics provided by this check.
+
+By default the SingleStore integration only collects metrics from the `MV_GLOBAL_STATUS`, `AGGREGATORS` and `LEAVES` tables.
+To collect additional system level metrics (cpu, disk, network io and memory), you can enable the following configuration option in the `singlestore.d/conf.yaml` file:
+
+```yaml
+    collect_system_metrics: true
+```
```

Comment:
```suggestion
```

let's get rid of this and put it in the host/container config options, as suggested above